src/style.css

```diff
@@ -1,3 +1,60 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 50px;
+  background-color: #fff;
+  align-items: center;
+  min-width: 1200px;
```

Comment:
The `min-width: 1200px;` in the `.header` class might cause layout issues on smaller screens. Consider using a responsive design approach, such as media queries, to adjust the layout for different screen sizes.